classes/PrestaShopAutoload.php

```diff
@@ -57,11 +57,11 @@ class PrestaShopAutoload
 
     protected function __construct()
     {
-        $this->root_dir = _PS_CORE_DIR_ . '/';
+        $this->root_dir = dirname(__DIR__);
         $file = static::getCacheFileIndex();
         $stubFile = static::getStubFileIndex();
         if (@filemtime($file) && is_readable($file) && @filemtime($stubFile) && is_readable($stubFile)) {
-            $this->index = include $file;
+            $this->index = require $file;
```

Comment:
Why is this change needed?

Reply:
It makes sense to me to make it fail instead of issuing a warning. If the cache file exists, it better be valid.